examples/non-monorepo/pages/index.tsx

```diff
@@ -0,0 +1,71 @@
+import Head from "next/head";
```

Comment:
I'm getting the following warning when running `yarn turbo dev` and loading the page in browser:
```
dev: ready - started server on 0.0.0.0:3000, url: http://localhost:3000
dev: event - initial compilation 2.363ms
dev: [404] /_next/webpack-hmr (WebSocket)
dev: A non-turbopack next.js client is trying to connect.
dev: Make sure to reload/close any browser window which has been opened without --turbo.
dev: warning - [resolve] pages/index.tsx
dev:   Unsupported module
dev:   The module next/head is not yet supported
```

Reply:
This is expected for turbopack right now

Reply:
Okay, I wasn't 100%, but I'm now feeling like we shouldn't opt people into `--turbo` yet.